### PR TITLE
Use a dimension to note how the user opened squoosh

### PR DESCRIPTION
**Pull request by petele**
- Checks how Squoosh was started and uses a dimension to segment pageviews between standalone and browser.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -13,11 +13,19 @@ if (!('customElements' in self)) {
...
+  let displayMode = 'browser';
+  const mqStandAlone = '(display-mode: standalone)';
+  if (navigator.standalone || window.matchMedia(mqStandAlone).matches) {
+    displayMode = 'standalone';
+  }
+  // Setup analytics
   window.ga = window.ga || ((...args) => (ga.q = ga.q || []).push(args));
   ga('create', 'UA-128752250-1', 'auto');
   ga('set', 'transport', 'beacon');
+  ga('set', 'dimensionX', displayMode);
```
Comment (petele) [inline src/index.ts]: Need to update `dimensionX` to whatever dimension we end up using.

**Review by surma — APPROVED** [review]
I pushed a commit fixing the type error.

Code looks good modulo dimension fix. @petele back over to you.

@jakearchibald PTAL

File: src/index.ts (hunk 1 above)
Comment (petele) [reply src/index.ts]: Dimension has been setup!

**Comment by petele** [comment]
I've setup the dimension, so once it gets Jake's LGTM, it's good to go.

**Review by jakearchibald — APPROVED** [review]
LGTM

**Comment by surma** [comment]
Gonna merge once CI passes

**Pull request merged**